pages.ko/common/git-blame.md

```diff
@@ -0,0 +1,12 @@
+# git blame
+
+> 각각의 파일 라인에 커밋 해쉬와 마지막 작성자를 보여줍니다.
+> 더 많은 정보: <https://git-scm.com/docs/git-blame>.
+
+- 각각의 라인에 작성자의 이름과 커밋 해쉬가 포함된 파일 출력:
+
+`git blame {{file}}`
```

Comment:
Please translate the placeholders, like `file`, for get this easier for non-English speakers.

Reply:
Addressed and updated 👍 Thank you! @navarroaxel